channel.js

```diff
@@ -603,6 +604,11 @@ TChannel.prototype.makeSubChannel = function makeSubChannel(options) {
             }
         }
     }
+    if (options.hostsWatchFilePath) {
```

Comment:
Should `hostsWatchFilePath` actually be `peerFile`?

The option for providing a list of peers without a file is `peers`. I think the main theme of the top-level option is that we're providing a mechanism to load a list of peers from a file. The fact that we watch the file for changes is just a nice detail. It doesn't need to be exposed in the name.